### PR TITLE
Add api version

### PR DESCRIPTION
**Pull request by telday**
### What does this PR do?
Adds a file with the current api version to the repository. This file will help keep track of the current OpenAPI version which releases alongside Conjur.

#### Change log
- [ ] The CHANGELOG has been updated, or
- [x] This PR does not include user-facing changes and doesn't require a CHANGELOG update

#### Test coverage
- [ ] This PR includes new unit and integration tests to go with the code changes, or
- [x] The changes in this PR do not require tests

#### Documentation
- [ ] Docs (e.g. `README`s) were updated in this PR, and/or there is a follow-on issue to update docs, or
- [x] This PR does not require updating any documentation

#### API Changes
- [ ] The [OpenAPI spec](https://github.com/cyberark/conjur-openapi-spec) has been updated to meet new API changes (or an issue has been opened), or
- [x] The changes in this PR do not affect the Conjur API

### CODE REVIEW

**Comment by izgeri** [comment]
@telday can we also add this to the CHANGELOG `Added` section and link to this file in the CONTRIBUTING? both the section where you describe the openapi workflow and in the "releasing" section (I know you have a follow-up issue for the latter, but we should at least mention that this should be manually checked for now)

**Comment by telday** [comment]
@izgeri Should be all set now

File: app/API_VERSION [hunk 1]
```diff
@@ -0,0 +1 @@
+5.1.0
```
Comment (izgeri) [inline app/API_VERSION:1]: this should just be at the root dir level, right? like VERSION?

File: CHANGELOG.md [hunk 2]
```diff
@@ -9,6 +9,10 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 ### Security
 - Upgrade github-pages in docs/Gemfile to resolve CVE-2021-28834 in kramdown dependency [cyberark/conjur#2099](https://github.com/cyberark/conjur/issues/2099)
 
+### Added
+- New File `API_VERSION` containing the current Conjur API Version to the repository.
```
Comment (izgeri) [inline CHANGELOG.md:13]: we need to link this to the OpenAPI spec - maybe change it to
```suggestion
- File `API_VERSION` containing the current Conjur API Version, which corresponds to a
  release of the [Conjur OpenAPI Spec](https://github.com/cyberark/conjur-openapi-spec).
```

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -510,6 +512,7 @@ version.
 1. Add a new, empty _Unreleased_ section to the changelog.
    - Remember to update the references at the bottom of the document.
 1. Change VERSION file to reflect the change. This file is used by some scripts.
+1. Change the API_VERSION file to reflect the new API version if there has been an update to the API.
```
Comment (izgeri) [inline CONTRIBUTING.md:515]:
```suggestion
1. Change the API_VERSION file to reflect the correct
   [OpenAPI spec release](https://github.com/cyberark/conjur-openapi-spec/releases)
   if there has been an update to the API.
```

**Review by izgeri — CHANGES_REQUESTED** [review]
a couple of minor suggestions for changes - also want to flag for @alexkalish

File: CHANGELOG.md [hunk 4]
```diff
@@ -9,6 +9,10 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 ### Security
 - Upgrade github-pages in docs/Gemfile to resolve CVE-2021-28834 in kramdown dependency [cyberark/conjur#2099](https://github.com/cyberark/conjur/issues/2099)
 
+### Added
+- File `API_VERSION` containing the current Conjur API Version, which corresponds to a
+  release of the [Conjur OpenAPI Spec](https://github.com/cyberark/conjur-openapi-spec).
```
Comment (izgeri) [inline CHANGELOG.md:14]: this also needs the link to the PR: [PR cyberark/conjur#2107](https://github.com/cyberark/conjur/pull/2107)

**Review by izgeri — APPROVED** [review]

**Pull request merged**